README.md

```diff
@@ -300,6 +300,24 @@ Self-hosted open source devops and collaboration tools, suitable for enterprise.
 - [Metabase](https://www.metabase.com/) - [🛠](https://stackshare.io/metabase) - [🐙](https://github.com/metabase/metabase) - An open-source business intelligence tool
 
 
+## Production Ready Django + Docker [↗](https://awesomestacks.dev/production-ready-django-docker)
+
+Bleeding edge django template focused on code quality and security.
+
+Build, test, and deploy pipelines are configured by default.
+
+- [Python3](https://www.python.org/) - [🛠](https://stackshare.io/python) - [🐙](https://github.com/cpython/python) - A programming language that lets you work quickly.
```

Comment:
1. Please change "Python3" to "Python 3"
1. Please fix the GitHub link to `https://github.com/python/cpython`